EmailNotificationActivity/index.ts

```diff
@@ -32,6 +31,7 @@ import { NonEmptyString } from "italia-ts-commons/lib/strings";
 import { Millisecond } from "italia-ts-commons/lib/units";
 import nodemailerSendgrid = require("nodemailer-sendgrid");
 
+import { MailUpTransport } from "io-functions-commons/dist/src/mailer/transports";
```

Comment:
You actually don't want to manually create a transport. Instead, we just want to call the `mailer`'s factory metodo for that (see [here](https://github.com/pagopa/io-functions-commons/pull/112#pullrequest-500583371-permalink:~:text=The%20folder%20has%20a%20index.ts%20entrypoint,proxied%20by%20the%20index%20module%20itself.)).

```suggestion
import { getMailerTransporter } from "io-functions-commons/dist/src/mailer";
```